.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
```

Comment:
Can we get rid of this and set an appropriate label instead?

Reply:
I'm 👎 on auto-setting labels without triage by the core team. I'm indifferent about the title, though note that it is common across many projects, even with labeling.